### PR TITLE
add confirmation screenshot

### LINKED ISSUE #3962: Early Access: Custom Certificates on Pantheon's Global CDN Doc Update
Re: [Early Access: Custom Certificates on Pantheon's Global CDN](https://pantheon.io/docs/custom-certificates/)

Priority (Low‚ Medium‚ High): Med

## Issue Description:

Doc doesn't state that when a custom certificate covers a domain, the HTTPS pane in the domain's detail view will indicate such along with the certificate's expiration date

## Suggested Resolution

Include that info with example screenshot

### PR DESCRIPTION
**Pull request by alexfornuto**
Closes #3962 

## Effect
PR includes the following changes:
- New step showing custom cert confirmation in the dashboard.

## Remaining Work
- [x] Review from @ari-gold 
- [ ] Copy Review

## Post Launch
To be completed by the docs team upon merge: 
- [ ] Update Status Report
- [ ] Archive from **Done** in Waffle

### CODE REVIEW

**Comment by alexfornuto** [comment]
@ari-gold BAM: https://3962https-static-docs.pantheonsite.io/docs/custom-certificates/#option-2-manually-managed-custom-certificates

File: source/_docs/custom-certificates.md [hunk 1]
```diff
@@ -57,12 +57,15 @@ If you require a custom, dedicated certificate, you can now bring it the Global
 
     Be sure to send these as separate files, not a "chained cert".
 
-4. [Test Before Going Live](#test-before-going-live) (optional, recommended)
+4. Once the certificate is in place, you will see under **Details** for your domain the following:
```
Comment (ari-gold) [inline source/_docs/custom-certificates.md]: "...for your domain(s)"

File: source/_docs/custom-certificates.md [hunk 2]
```diff
@@ -57,12 +57,15 @@ If you require a custom, dedicated certificate, you can now bring it the Global
...
 
-5.  <a href="#disable-lets-encrypt-with-caa-records-required" data-proofer-ignore>Disable Let's Encrypt by adding CAA DNS records</a>.
+    ![Custom Certificate Confirmation](/source/docs/assets/images/dashboard/custom-cert-confirm.png)
 
-6. Update `A` and `AAAA` records provided by Pantheon Support.
+5. [Test Before Going Live](#test-before-going-live) (optional, recommended)
 
+6.  <a href="#disable-lets-encrypt-with-caa-records-required" data-proofer-ignore>Disable Let's Encrypt by adding CAA DNS records</a>.
+
+7. Update `A` and `AAAA` records provided by Pantheon Support.
```
Comment (ari-gold) [inline source/_docs/custom-certificates.md]: scope creep, something like:
"...by Pantheon Support. Note that even for subdomains A and AAAA records are required. Do not use a CNAME."
